manifests/input.yaml

```diff
@@ -67,29 +67,29 @@ work/input/RYF.vas.1990_1991.nc:
     binhash: 17224eecb745b6ce72604b41201c1fe7
     md5: 88dc8c70338bbc8f5efc48ed0009a99f
 work/input/access-om2-1deg-ESMFmesh.nc:
-  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/access-om2-1deg-ESMFmesh.nc
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/share/access-om2-1deg-ESMFmesh.nc
   hashes:
-    binhash: db1ebe07724c4b9350947423239e58f0
+    binhash: b76768044950ac2a165e3c2cd70e2f56
     md5: 670568bbb1f7f14c534b37ec895af118
 work/input/access-om2-1deg-nomask-ESMFmesh.nc:
-  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/access-om2-1deg-nomask-ESMFmesh.nc
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/share/access-om2-1deg-nomask-ESMFmesh.nc
   hashes:
-    binhash: 465e2826fc0b9344eda8bcde953160a9
+    binhash: 25269214daf2808dbf536f434b707b71
     md5: d40d8512d683a089b6feb343d903ad71
 work/input/grid.nc:
-  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/grid.nc
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/share/grid.nc
```

Comment:
This should be in the `cice` directory (you've probably already moved this for https://github.com/COSIMA/MOM6-CICE6-WW3/pull/12)

Reply:
Good catch. Fixed.